src/index.html

```diff
@@ -7,11 +7,157 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
+    <title>Html From</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="container-info">
+        <legend>Personal information</legend>
+        <div class="container">
+          <label for="surname">Surname:</label>
+          <input type="text"
+          name="surname"
+          id="surname"
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="surname"
            id="surname"
          >
```
[checklist p.3](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

fix code style all over the page

Reply:
okey, thanks, after morning lesson i will fix it